circuits/ts/__tests__/CalculateTotal.test.ts

```diff
@@ -1,16 +1,16 @@
-import { expect } from "chai";
-import tester from "circom_tester";
+import { WitnessTester } from "circomkit";
```

Comment:
maybe we only need the type here? `import { type WitnessTester } from "circomkit"`